packages/example/src/components/VerifyMessage/VerifyMessage.tsx

```diff
@@ -1,31 +1,63 @@
-import { Card, Form, Input } from 'antd';
+import { Button, Card, Form, Input, message as antdMessage } from 'antd';
+import { useEffect, useState } from 'react';
 import { formLayout } from '../utils';
+import { useSnap } from '../../hooks/useSnap';
 
 export function VerifyMessage(): JSX.Element {
+  const { verifyMessage, isMessageVerified } = useSnap();
+  const [message, setMessage] = useState('');
+  const [signature, setSignature] = useState('');
+  const [messageApi, contextHolder] = antdMessage.useMessage();
+
+  useEffect(() => {
+    if (isMessageVerified === true) {
+      void messageApi.success('Successfully verified');
+    }
+    if (isMessageVerified === false) {
+      void messageApi.error('Unable to verify this message and signature');
+    }
+  }, [isMessageVerified]);
```

Comment:
what is with a case if the message two different messages are verified with successful results?

Reply:
you'r right, will add this in next pr
` onAnimationStart={() => {resetIsMessageVerified()}`